README.md

```diff
@@ -363,3 +363,17 @@ the issue, see <https://github.com/dependabot/cli/issues/113#issuecomment-161012
 [smoke-tests]: https://github.com/dependabot/smoke-tests/tree/main/tests
 [dependabot-updater-job]: https://github.com/dependabot/dependabot-core/blob/main/updater/lib/dependabot/job.rb
 [PAT]: https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token
+
+### Update/Rebuild cli
```

Comment:
Thanks for this, to snap to the pattern in the troubleshooting aspects, I think the header should be the error you got, and then the body should describe how to fix it.